wordpress-mobile/WordPress-Aztec-iOS
# Attachments: Interface

#### Details:
In this PR we're adding a super simple ViewController, that allows us to refresh both, Sizing and Alignment properties.

#### Testing:
1. Add a new image
2. Tap on it
3. Verify that the UI properly reflects any of the new settings you might select

Please, also verify that the Gestures Recognizer doesn't lock scroll events, even when performed on top of the image.

Needs Review: @diegoreymendez 
Thanks in advance!
